### PR TITLE
Bug 1960089: CamelCase ImageStream related pages and breadcrumb

### PR DESCRIPTION
**Pull request by yapei**
CamelCased ImageStream related pages and breadcrumb links
Fixes https://bugzilla.redhat.com/show_bug.cgi?id=1960089

ImageStreams list page
Screen Shot 2021-05-13 at 3 40 56 PM

ImageStream details page
Screen Shot 2021-05-13 at 3 41 09 PM

ImageStreamTag details page
Screen Shot 2021-05-13 at 3 41 26 PM

### CODE REVIEW

**Comment by yapei** [comment]
`Alertmanager: Slack Receiver Form: saves globals and advanced fields correctly` test flake, rerun

**Comment by yapei** [comment]
@spadgett Could you help review?

File: frontend/public/components/image-stream-tag.tsx [hunk 1]
```diff
@@ -55,9 +55,9 @@ export const ImageStreamTagsDetails: React.SFC<ImageStreamTagsDetailsProps> = ({
       <div className="co-m-pane__body-group">
         <div className="row">
           <div className="col-md-6 col-sm-12">
-            <SectionHeading text="Image Details" />
+            <SectionHeading text="ImageStreamTag details" />
```
Comment (spadgett) [inline frontend/public/components/image-stream-tag.tsx:58]: @yapei Do you mind opening a bug to translate these properties? We can do it as a separate bug. It looks like we missed these.

**Review by spadgett — APPROVED** [review]
/lgtm

File: frontend/public/components/image-stream-tag.tsx (hunk 1 above)
Comment (yapei) [reply frontend/public/components/image-stream-tag.tsx:58]: @spadgett  I opened [one](https://bugzilla.redhat.com/show_bug.cgi?id=1960102)

**Pull request merged**